src/main.ts

```diff
@@ -46,13 +60,21 @@ export const playLevel = async (levelUrl: string, videoName: string) => {
   // stop video recording
   const gamplayVideoUri = await recorder.stop();
 
+  console.log("Grabbing expression...")
+
   // get results
   const expression = await page.evaluate(
     "world.level.ui.mathField.getPlainExpression()"
   );
+
+  console.log("Grabbing score...")
+
   const T = await page.evaluate(
-    "parseFloat(world.level.ui.runButton.innerText.trim().split('=')[1])"
+    "parseFloat(world.level.ui.completionTime.innerText)"
```

Comment:
@JosiasAurel Bringing your attention specifically to this, which resolves the issue.

Note that this will not be available from production sinerider until https://github.com/hackclub/sinerider/pull/273 is landed.  Until then you can use https://sinerider-76irjc1pq.hackclub.dev/

Reply:
great the PR was merged as well